app/assets/javascripts/orders.coffee

```diff
@@ -40,15 +40,18 @@ findLastCategory = ->
   orders = $('.well').find '.order'
   $(orders[orders.length-1]).find '#category'
 
-$(document).on "change", ".form-control", (e) ->
+$(document).on "change", ".orders-list .form-control", (e) ->
   $(e.target).closest("form").submit()
 
-$(document).on "click", ".add-item", (e) ->
+$(document).on "click", ".edit-order-form .add-item", (e) ->
   e.preventDefault()
   e.stopPropagation()
   $("#add_inventory_modal").modal("show")
 
-$(document).on "click", "#add-item-row", (event) ->
+$(document).on "click", ".orders-list tr.order", (e) ->
+  orderRowClicked(e, $(e.target).closest('tr.order'))
```

Comment:
Doing the above should make this click handler unnecessary... it is handled by `data-click-fn`.